doc/themes/scikit-learn-modern/static/css/theme.css

```diff
@@ -0,0 +1,1096 @@
+}
+
+div.admonition p, div.admonition dl, div.admonition dd {
+  margin-bottom: 0
+}
+
+div.deprecated {
+  color: #b94a48;
+  background-color: #F3E5E5;
+  border: 1px solid #eed3d7;
+}
+
+div.seealso {
+  background-color: #FFFBE8;
+  border: 1px solid #fbeed5;
+  color: #AF8A4B;
+}
+
+div.versionchanged {
+  margin-top: 0.5rem;
+  padding: 0.5rem;
+  background-color: #FFFBE8;
+  border: 1px solid #fbeed5;
+  border-radius: 0.5rem;
+}
+
+div.versionchanged p {
+  margin-bottom: 0;
+}
+
+dt.label {
+  float: left;
+  padding-right: 0.5rem;
+}
+
+/* copy buttonn */
+div.highlight:hover span.copybutton {
+  background-color: #3F556B;
+  color: white;
+}
+
+div.highlight:hover span.copybutton:hover {
+    background-color: #20252B;
+}
+
+div.body img.align-center {
+  max-width: 800px;
+}
+
+div.body img {
+    max-width: 100%;
+    height: unset!important; /* Needed because sphinx sets the height */
+}
+
+img.align-center, .figure.align-center, object.align-center {
+  display: block;
+  margin-left: auto;
+  margin-right: auto;
+  margin-bottom: 1rem;
+  text-align: center;
+}
+
+img.align-right, .figure.align-right, object.align-right {
+  clear: right;
+  float: right;
+  margin-left: 1em;
+}
+
+a.brackets::after, span.brackets > a::after {
+  content: "]";
+}
+
+a.brackets::before, span.brackets > a::before {
+    content: "[";
+}
+
+/* copybutton */
+
+.copybutton {
+  cursor: pointer;
+  position: absolute;
+  top: 0px;
+  right: 0px;
+  border: 1px solid rgb(221, 221, 221);
+  color: rgb(221, 221, 221);
+  font-family: monospace;
+  padding-left: 0.2rem;
+  padding-right: 0.2rem;
+}
+
+div.highlight:hover span.copybutton::after {
+  background: #3F556B;
+  border-radius: 0.25rem;
+  color: white;
+  content: attr(title);
+  padding: 0.25rem;
+  position: absolute;
+  z-index: 98;
+  width: 100px;
+  font-size: 0.7rem;
+  top: 0;
+  right: 0;
+}
+
+/* world */
+
+img.avatar {
+  width: 100%;
+}
+
+/* table */
+table.align-default {
+  margin-left: auto;
+  margin-right: auto;
+}
+
+table.docutils tr:nth-child(odd) {
+  background-color: #F0F7FA;
+}
+
+table.docutils tr {
+  border-style: solid none solid none;
+  border-width: 1px 0;
+  border-color: #ddd;
+}
+
+table.docutils td, table.docutils th {
+  padding: 0.125rem 0.5rem 0.125rem 0.25rem;
+}
+
+table.docutils {
+  margin-bottom: 1rem;
+  line-height: 1rem;
+  max-width: 100%;
+}
+
+table.docutils p {
+  margin-bottom: 0;
+}
+
+table.docutils td:last-child p {
+  white-space: pre-wrap;
+  word-wrap: break-word;
+}
+
+/* gallery */
+
+div.sphx-glr-thumbcontainer {
+  min-height: 250px;
+  font-size: 0.9rem;
+}
+
+.sphx-glr-example-title > :target::before {
+  display: block;
+  content: "";
+  margin-top: -150px;
+  height: 150px;
+  visibility: hidden;
+}
+
+.sphx-glr-script-out .highlight pre {
+  padding-top: 1ex;
+  padding-right: 1ex;
+}
+
+.sphx-glr-script-out div.highlight {
+  padding: 0;
+}
+
+
+@media screen and (min-width: 1540px) {
+  .sphx-glr-download-link-note {
```

Comment:
Great stuff! I just noticed that the note about the binder link is floated right, I wish I had the web skills to think of this in the first place and then execute it as well.
![image](https://user-images.githubusercontent.com/1680079/67147508-cc1cd000-f295-11e9-9ed6-36089903c934.png)